tools/clang/lib/CodeGen/CGExprAgg.cpp

```diff
@@ -741,10 +741,13 @@ void AggExprEmitter::VisitCastExpr(CastExpr *E) {
         LValue LV;
         if (DeclRefExpr *SrcDecl = dyn_cast<DeclRefExpr>(Src))
           LV = CGF.EmitLValue(SrcDecl);
-        else if (ArraySubscriptExpr *ArraySubExpr = dyn_cast<ArraySubscriptExpr>(Src))
+        else if (ArraySubscriptExpr *ArraySubExpr =
+                     dyn_cast<ArraySubscriptExpr>(Src))
```

Comment:
Is this formatting-only change on purpose?

Reply:
No. I'll revert it.